src/resources/views/crud/fields/upload.blade.php

```diff
@@ -164,6 +166,9 @@ function bpFieldInitUploadElement(element) {
 
                     // add a hidden input with the same name, so that the setXAttribute method is triggered
                     $("<input type='hidden' name='"+fieldName+"' value=''>").insertAfter(fileInput);
```

Comment:
Hello @promatik 

I kind of did the same trick for the select multiples (if empty browser does not send them), so I switched between single (when empty so it gets posted) and back to multiple when user selects something. 

So with this change, the hidden input might not be needed? From my understanding the hidden input is there to solve the same problem you solved with field switching. 

Let me know.

Reply:
@pxpm TL;DR yes I think the hidden input isn't needed.

I'm not quite sure what the current hidden input is doing, because it's not printed when you edit the entry. Right now when you're editing an entry this is generated: `<input type="file" value="file.pdf" class="file_input backstrap-file-input" />`. This doesn't work because you can't set the value of `<input type="file">` for security reasons. So the field is not submitted on an update.

Reply:
@promatik I am trying to get around this and I feel like I am trying to catch myself here. Help me out :)

When you don't sent nothing in request means the `mutator` would not run.

If we consistently send the `value` even if not changed, it means that the mutator would run everytime right ?

So for example, I create an entry and add 1 file. I then go to edit that entry and change `name` but don't touch in file. (name is posted, file is  not posted). With this change we will post `file` with the same value we have in database right? And that would fire the mutator again ?